_components/typography/typography.md

```diff
@@ -50,7 +50,7 @@ Controlling the length of lines of text (also known as _measure_) helps readers
 
 {:.usa-content-list}
 - **Most lines of text should be 45–90 characters.** The current standard range for readable line length is 45 ([measure 1]({{ site.baseurl }}/design-tokens/typesetting/measure/){:.token}) to 90 characters per line ([measure 5]({{ site.baseurl }}/design-tokens/typesetting/measure/){:.token}). A good target for long texts is 66 characters ([measure 2]({{ site.baseurl }}/design-tokens/typesetting/measure/){:.token}).
-- **Text with greater line height can have a longer measure.** Since of the function of measure is to help readers move from one line of text to another more naturally, the effects of a long measure can be lessened by increasing the space between lines. Thus, text with more space between lines can have somewhat longer line length.
+- **Text with greater line height can have a longer measure.** Since one of the function of measure is to help readers move from one line of text to another more naturally, the effects of a long measure can be lessened by increasing the space between lines. Thus, text with more space between lines can have somewhat longer line length.
```

Comment:
Small typo 
```suggestion
- **Text with greater line height can have a longer measure.** Since one of the functions of measure is to help readers move from one line of text to another more naturally, the effects of a long measure can be lessened by increasing the space between lines. Thus, text with more space between lines can have somewhat longer line length.
```